cypress/integration/visitorCanNavigateThroughCategories.feature.js

```diff
@@ -22,7 +22,20 @@ describe("A user visiting the application can display articles by categories", (
     it("is expected to see a collection of articles", () => {
       cy.get("[data-cy=article-collection]")
         .children()
-        .should("have.length", 20);
+        .should("have.length", 4);
+    });
+
+    it("is expected to see teaser teaser", () => {
```

Comment:
This test is redundant. You are checking for the same thing in test that starts on line 52